### ISSUE TITLE
FileNotFoundException on test-osuosl-centos74-ppc64le-1

### ISSUE THREAD

**Issue by adamfarley** [AdoptOpenJDK/openjdk-infrastructure#1878]
TLDR: Please restart the jenkins agent on test-osuosl-centos74-ppc64le-1

Details:

```
java.io.FileNotFoundException: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.275.b01-0.el7_9.ppc64le/jre/lib/resources.jar
	at sun.misc.URLClassPath$JarLoader.getJarFile(URLClassPath.java:938)
	at sun.misc.URLClassPath$JarLoader.access$800(URLClassPath.java:801)
	at sun.misc.URLClassPath$JarLoader$1.run(URLClassPath.java:886)
	at sun.misc.URLClassPath$JarLoader$1.run(URLClassPath.java:879)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.misc.URLClassPath$JarLoader.ensureOpen(URLClassPath.java:878)
	at sun.misc.URLClassPath$JarLoader.getResource(URLClassPath.java:1050)
Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to test-osuosl-centos74-ppc64le-1
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1800)
		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:357)
		at hudson.remoting.Channel.call(Channel.java:1001)
```

https://ci.adoptopenjdk.net/job/Test_openjdk8_j9_sanity.openjdk_ppc64le_linux/455/
https://trss.adoptopenjdk.net/output/test?id=601138418a00d84d71f746c5

- test suite/name: sanity.openJDK (Post step)
- Is there an existing issue elsewhere covering this: I think this is another case of #1468 .
- Which machine(s) does it work on? - All other machines, I think.
- Which machine(s) does it fail on? - test-osuosl-centos74-ppc64le-1
- Do you have a link to a Grinder re-run if the test with the failure?
https://ci.adoptopenjdk.net/job/Grinder/parambuild/?TARGET=sanity.openjdk&SDK_RESOURCE=upstream&TEST_FLAG=&UPSTREAM_TEST_JOB_NAME=&DOCKER_REQUIRED=false&ACTIVE_NODE_TIMEOUT=0&EXTRA_DOCKER_ARGS=&VENDOR_TEST_DIRS=&TKG_OWNER_BRANCH=AdoptOpenJDK%3Amaster&PLATFORM=ppc64le_linux&KEEP_REPORTDIR=true&PERSONAL_BUILD=false&ADOPTOPENJDK_REPO=https%3A%2F%2Fgithub.com%2FAdoptOpenJDK%2Fopenjdk-tests.git&LABEL=test-osuosl-centos74-ppc64le-1&EXTRA_OPTIONS=&CUSTOMIZED_SDK_URL=&BUILD_IDENTIFIER=&ADOPTOPENJDK_BRANCH=master&ARTIFACTORY_SERVER=&KEEP_WORKSPACE=false&JDK_VERSION=8&USER_CREDENTIALS_ID=&ITERATIONS=1&VENDOR_TEST_REPOS=&JDK_REPO=https%3A%2F%2Fgithub.com%2Fibmruntimes%2Fopenj9-openjdk-jdk8&RELEASE_TAG=&OPENJ9_BRANCH=master&OPENJ9_SHA=&JCK_GIT_REPO=&VENDOR_TEST_BRANCHES=&UPSTREAM_JOB_NAME=build-scripts%2Fjobs%2Fjdk8u%2Fjdk8u-linux-ppc64le-openj9&OPENJ9_REPO=https%3A%2F%2Fgithub.com%2Feclipse%2Fopenj9.git&CUSTOM_TARGET=&VENDOR_TEST_SHAS=&JDK_BRANCH=openj9&LABEL_ADDITION=&ARTIFACTORY_REPO=&ARTIFACTORY_ROOT_DIR=&UPSTREAM_TEST_JOB_NUMBER=&DOCKERIMAGE_TAG=&JDK_IMPL=openj9&SSH_AGENT_CREDENTIAL=&AUTO_DETECT=true&TKG_SHA=&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=&NUM_MACHINES=&OPENJDK_SHA=&BUILD_LIST=openjdk&UPSTREAM_JOB_NUMBER=886&TIME_LIMIT=10&JVM_OPTIONS=&PARALLEL=None

Any other details:
In #1468, Will said:
```
I theorise that somebody updated JDK8 without restarting the Jenkins agent
```
and that seemed to fix the issue there.

So I ask that we start by restarting the jenkins agent on this machine and seeing if that fixes the problem.

**Comment by Haroon-Khel** [comment]
Ive restarted the jenkins agent @adamfarley

**Comment by adamfarley** [comment]
Thanks @Haroon-Khel 

Running - https://ci.adoptopenjdk.net/job/Grinder/6199/console

**Comment by Haroon-Khel** [comment]
I think the rerun will need to use a nightly sdk instead of an upstream one, as it is having trouble copying over the artefacts from upstream job youve pointed it to

**Comment by sxa** [comment]
I think I saw Adam walking down the road about half an hour ago so he may not have read this ;-) I've reinitiated it with the latest build: https://ci.adoptopenjdk.net/job/Grinder/6204/console

**Comment by adamfarley** [comment]
Thanks @sxa

**Comment by adamfarley** [comment]
The issue still seems to be happening.

https://trss.adoptopenjdk.net/output/test?id=601b1da9ddf93e76e9fecd79

/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.275.b01-0.el7_9.ppc64le/jre/lib/resources.jar

Seems to still be looking for the same file + jdk8 version.

Could someone please check and see if that file is actually there?

**Comment by Haroon-Khel** [comment]
Restarted the agent again. Rerunning at https://ci.adoptopenjdk.net/job/Grinder/6454/console

**Comment by Haroon-Khel** [comment]
The latest grinder passed. I think I didnt restart the agent correctly the first time. Ive done it correctly now. Closing
